### DISCUSSION TITLE
DAPP web 3

### DISCUSSION THREAD

**Discussion by Alien69Flow** [community/community#136667] [Enterprise]
### Select Topic Area

Question

### Body

como hago para rehacer repositorios de una dapp web 3 como la mia ? AlienFlow.Space esto hice en wordpress, esto se hizo en AlienFlow.vercel.app pero nos dejo tirado sin acceso a repositorios y ahora como hago para rehacerlos de nuevo directamente aqui? react, next, node? vercel? necesito aclararme, por favor, Gracias

**Comment by ebndev** [comment]
Hi @AitorAilen, thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting to ensure discussions are adhering to our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md). 

We’ll be locking this post for now, but please feel encouraged to create a new discussion in English if you still need support.
